File: numeros.go
package main

import (
	"fmt"
	"time"
)

func main() {
	// Pedimos al usuario que ingrese los dos números
	var num1, num2 int
	fmt.Print("Ingrese el primer número: ")
	fmt.Scanln(&num1)
	fmt.Print("Ingrese el segundo número: ")
	fmt.Scanln(&num2)

	suma := 0 // Inicializamos la variable suma

	// Registro del tiempo de inicio de la ejecución
	inicio := time.Now()

	// Suma consecutiva de los números entre num1 y num2
	for i := num1; i <= num2; i++ {
		suma += i // Acumulación de la suma de los números
	}

	// Registro del tiempo de finalización de la ejecución
	fin := time.Now()

	// Cálculo del tiempo de ejecución en segundos
	tiempoEjecucion := fin.Sub(inicio).Seconds()

	// Mostrar la suma y el tiempo de ejecución en la consola
	fmt.Printf("La suma de los números entre %d y %d es: %d\n", num1, num2, suma)
	fmt.Printf("Tiempo de ejecución: %.6f segundos\n", tiempoEjecucion)
}

File: numeros.py
# Importamos el módulo necesario para leer la entrada del usuario
import time

# Pedimos al usuario que ingrese los dos números
num1 = int(input("Ingrese el primer número: "))
num2 = int(input("Ingrese el segundo número: "))

suma = 0  # Inicializamos la variable suma

# Registro del tiempo de inicio de la ejecución
inicio = time.time()

# Suma consecutiva de los números entre num1 y num2
for i in range(num1, num2 + 1):
    suma += i  # Acumulación de la suma de los números

# Registro del tiempo de finalización de la ejecución
fin = time.time()

# Cálculo del tiempo de ejecución en segundos
tiempo_ejecucion = fin - inicio

# Mostrar la suma y el tiempo de ejecución en la consola
print("La suma de los números entre", num1, "y", num2, "es:", suma)
print("Tiempo de ejecución:", tiempo_ejecucion, "segundos")